src/firefly/java/edu/caltech/ipac/util/RegionFactory.java

```diff
@@ -568,15 +569,19 @@ private static RegionValue convertToRegionValue(String radius_string, ValueType
                     unit= RegionValue.Unit.ARCMIN;
                     break;
                 case 'd':      // degrees
+                case 'D':
```

Comment:
why not convert `unit_char` to lower as well.

`Character.toLowerCase()` static method in the java.lang.Character package.